### ISSUE TITLE
Specify size of individual plots

### ISSUE THREAD

**Issue by cstjean** [JuliaPlots/Plots.jl#1431]
I often want to do N plots, each with a "reasonable" size. I would have thought that this code would work: 

```julia
N = 7
plot([plot([1,2], size=(300, 400)) for i in 1:N]...)
```

But the `size` attribute is used as the size of the total figure.

**Comment by mkborregaard** [comment]
Yes you can't do that at the moment - filing as a feature request
